### PR TITLE
bears/css: Add CSSCombBear

### PR DESCRIPTION
**Pull request by yash-nisar**
Closes https://github.com/coala/coala-bears/issues/634

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by yash-nisar** [comment]
CC @Nosferatul @Makman2

File: tests/css/csscomb_test_files/.csscomb.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "space-after-opening-brace": 1,
+  "space-before-closing-brace": 1
+}
```
Comment (Makman2) [inline tests/css/csscomb_test_files/.csscomb.json]: no newline on purpose?

Comment (yash-nisar) [inline tests/css/csscomb_test_files/.csscomb.json]: Somewhere at the back of my mind I had the newline fear, but failed to review myself. My bad.

File: bears/css/CSSCombBear.py [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+                        use_space_between_declarations: int=None,
+                        use_space_after_opening_brace: int=None,
+                        use_space_before_closing_brace: int=None,
+                        use_space_after_selector_delimiter: int=1,
+                        use_space_before_selector_delimiter: int=1,
+                        prohibit_trailing_whitespace: bool=True,
+                        prohibit_units_in_zero_valued_dimensions: bool=True,
+                        vendor_prefix_align: bool=None,
+                        use_lines_between_rulesets: int=1,
+                        csscomb_config: str=''):
```
Comment (Makman2) [inline bears/css/CSSCombBear.py]: `csscomb_config` is not documented

Comment (Makman2) [inline bears/css/CSSCombBear.py]: nvm

File: tests/css/CSSCombBearTest.py [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+from coalib.settings.Section import Section
+from coalib.settings.Setting import Setting
+from coalib.testing.LocalBearTestHelper import LocalBearTestHelper
+from coalib.testing.BearTestHelper import generate_skip_decorator
+
+from bears.css.CSSCombBear import CSSCombBear
+
+
+@generate_skip_decorator(CSSCombBear)
+class ArtisticStyleBearTest(LocalBearTestHelper):
```
Comment (Makman2) [inline tests/css/CSSCombBearTest.py]: test name ;)

**Comment by Makman2** [comment]
you don't have test files, so you don't need to place a `__init__` into a test directory :+1:

File: tests/css/CSSCombBearTest.py (hunk 3 above)
Comment (yash-nisar) [inline tests/css/CSSCombBearTest.py]: Haha, the copy paste bug.

**Comment by yash-nisar** [comment]
Updated @Makman2

**Comment by Nosferatul** [comment]
nice @yash-nisar 👍 documented, clean code.

**Comment by Makman2** [comment]
ack 2bef3df

**Comment by Makman2** [comment]
ack 2bef3df

**Comment by Makman2** [comment]
@rultor merge

**Pull request merged**